src/søknad/barna-dine/LeggTilBarn.less

```diff
@@ -0,0 +1,71 @@
+
+.legg-til-barn {
+    .fødselsnummer {
+        display: flex;
+        flex-direction: row;
+        justify-content: flex-start;
+        margin-top: 3rem;
+    }
+
+    .bo-hos-deg-spørsmål {
+        margin-top: 2rem;
+    }
+
+    .legg-til-barn-knapp {
+        display: block;
+        margin: 0 auto;
+        margin-top: 5rem;
+    }
+
+    .personnummer {
+        margin-left: 2rem;
+    }
+
+    .termindato {
+        margin-top: 4rem;
+    }
+
+    h2 {
```

Comment:
Skal denne gjelde hele prosjektet? Isåfall bør den kanskje overstyres i index.less?

Reply:
Tror ikke vi kan anta det, skal gjelde under `.legg-til-barn`